syntax = "proto3";

message PersonMessage {
   int32 id = 1;
   string first_name = 2;
   string last_name = 3;
   string company_name = 4;
}

message LocationMessage {
    int32 person_id = 1;
    string creation_time = 2;
    string latitude = 3;
    string longitude = 4;
}
message Person {
    bool person = 1;
}

message Location {
    bool location = 1;
}
service UdaConnectService {
    rpc createPerson(PersonMessage) returns (Person);
    rpc createLocation(LocationMessage) returns (Location);
}